SMLHelper/Utility/EnumCacheManager.cs

```diff
@@ -193,11 +199,38 @@ internal void LoadCache()
                 Logger.Error($"Caught exception while reading cache!{Environment.NewLine}{exception}");
             }
 
+            try
+            {
+                string savePathDir = GetDeactivatedCachePath();
+                
+                if (!File.Exists(savePathDir))
+                {
+                    cacheLoaded = true;
+                    return;
+                }
+
+                string[] allText = File.ReadAllLines(savePathDir);
+                foreach (string line in allText)
+                {
+                    string[] split = line.Split(':');
+                    string name = split[0];
+                    string index = split[1];
+
+                    entriesFromDeactivatedFile.Add(Convert.ToInt32(index), name);
+                    entriesFromFile.Add(Convert.ToInt32(index), name);
```

Comment:
Did you intend to load the deactivated items into the `entriesFromFile` collection?

Also, this chunk of code is duplicated for each file and could be refactored.

Reply:
Yes i loaded it into the entries from file because that is what allows it to not reuse those ID's and is what allows you to add mods back into the game..    although they will need to add the mod then save and reload.

Reply:
unless i add a check somewhere that when things are going into the request list it check the deactivated list and removes it from the deactivate list that way they would not need to reload

Reply:
but cross mod support would still require reloading

Reply:
actually no it would work without reload

Reply:
if you want we could actually keep it all in one file if we add a third item per line and have it as a bool...

Reply:
Using two separate files is fine. It simplifies how the data is stored.